.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,30 @@
+name: PR build workflow 
+
+on:
+    pull_request:
+        branches:
+            - master
+            
+jobs:
+    build:
+        name: build
+        runs-on: ${{ matrix.os }}
+        
+        strategy:
+            fail-fast: false
+            matrix:
+                os: [windows-latest]
```

Comment:
Do we need unit test for .net core on `linux`  as well?

Reply:
Yup. We should add linux as well. Good call.